docs/project_droneworld/about.md

```diff
@@ -1,58 +1,54 @@
 
 ### Information
 
 - **Source Code:**  https://github.com/oss-slu/DroneWorld [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/DroneWorld)
-- **Client** [Dr.Ankit Agrawal](https://www.slu.edu/science-and-engineering/academics/computer-science/faculty-and-staff/ankit-agrawal.php) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/ankyAgrawal)
-- **Current Tech Lead:** [Yashaswini Shivalingaiah](https://yashaswinishivalingaiah.com/) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/yashaswini-slu)  [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/yashaswini-shivalingaiah-467a9652/)
+- **Client:** [Dr. Ankit Agrawal](https://www.slu.edu/science-and-engineering/academics/computer-science/faculty-and-staff/ankit-agrawal.php) [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/ankyAgrawal)
+- **Current Tech Lead:** Mohammad Al-Hanoosh [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/mohamdlog)  <!--[<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/)-->
 - **Developers:**
-    - Kaleb Yu [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/kalyus)
-    - Szymon Rostkowski [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/sr259)
-    - Hebron Bekele [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/hebronh) 
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/)
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/)
+    - Student [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/) 
 - **Start Date:** 9th Feb 2023
 - **Adoption Date:** 25 Aug 2023
-- **Technologies Used:**
-    - Tech Stack: Simulation, Testing, Visualization, Autonomous Systems
-    - Key Languages: Python, TypeScript with JavaScript
-    - Key Frameworks: React (Frontend), Flask (Backend APIs), Unreal Engine (Simulation)
-    - Databases: MongoDB 
-    - Critical Dependencies: 
-        - PX4 and Ardupilot
-        - Unreal Engine 
+
+### Technologies and Dependencies
+
+- **Key Languages:** Python, TypeScript, JavaScript
+- **Key Technologies:** React (Frontend), Flask (Backend APIs), Unreal Engine with AirSim (Simulation)
+- **Databases:** MongoDB
+- **Critical Dependencies:** PX4, ArduPilot, Unreal Engine
 - **Type:** Desktop
-- **License:**
+- **License:** [MIT](https://opensource.org/license/mit)
```

Comment:
The repo doesn't have a licence file. Check with Dr. Agrawal to confirm what license he's comfortable with. Please, cc oss@slu.edu on those e-mails so we have an institutional paper trail for decisions like this.

Reply:
I have a meeting with Dr. Agrawal next week, so I will discuss it with him then. In the meantime, I will revert the change.